proxy/api/src/process.rs

```diff
@@ -171,7 +150,9 @@ async fn run_rigging(
                         .await
                         .expect("Failed to read session store");
 
-                    if seeds == last_seeds {
+                    let current_status = peer_control.current_status().await;
+
+                    if seeds == last_seeds && current_status != coco::PeerStatus::Started {
```

Comment:
Shouldn’t we broadcast seeds when the status is `Offline`?

Reply:
You are right.